javascript-methods/main.js

```diff
@@ -0,0 +1,61 @@
+var num1 = 28;
+var num2 = 16;
+var num3 = 47;
+
+var maximumValue = Math.max(num1, num2, num3);
+console.log('The value of maximumValue: ', maximumValue);
+
+var heroes = ['Iron Man', 'Captain America', 'Thor', 'Hulk'];
+var randomNumber = Math.random();
+randomNumber += randomNumber * heroes.length;
```

Comment:
Line 10 is doing more than just multiplying `randomNumber` by the `length` of `heroes`, It's also adding that product to the previous value:

---

![image](https://user-images.githubusercontent.com/16143771/145268636-9ca77cde-1856-4b5a-a944-c3cb17ae89e2.png)